ajax/libs/web3/package.json

```diff
@@ -8,6 +8,7 @@
     "type": "git",
     "url": "https://github.com/ethereum/web3.js.git"
   },
+  "homepage": "https://github.com/ethereum/wiki/wiki/JavaScript-API",
```

Comment:
Let's add this line in the first commit.
Because it's better to make commit as commit message mention.

Thank you.